assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -250,17 +125,18 @@ const ChecklistTab = (props) => {
     (args) => {
       const { id, message, help, pageGroup } = args;
       const onPrepublish = getOnPrepublishSelect(args);
-      const accessibleText = __('Select offending element', 'web-stories');
       return (
-        <Row {...onPrepublish} tabIndex={0} key={id} pageGroup={pageGroup}>
-          {onPrepublish.onClick ? (
-            <Underline title={accessibleText}>{message}</Underline>
-          ) : (
-            message
-          )}
-          <HelperText>{help}</HelperText>
-          {onPrepublish.onClick && (
-            <VisuallyHidden>{accessibleText}</VisuallyHidden>
+        <Row key={id} pageGroup={pageGroup}>
+          <IssueTitle>{message}</IssueTitle>
+          <IssueDescription>{help}</IssueDescription>
+          {onPrepublish?.onClick && (
+            <ButtonAsLink
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+              aria-label={TEXT.ACCESSIBLE_LINK_TITLE}
+              {...onPrepublish}
```

Comment:
Nit: Not related to this pr at all so don't worry about it

This variable naming leads me to believe that this is an event handler since in react we prefix so many props/variables with `on`. I feel like this should be `prepublishProps` 🤔

Reply:
i agree and even broke this briefly because of that 😅  .... i will just fix it here.